### PR TITLE
refactor(configuration): minor configuration updates

### PR DESCRIPTION
**Pull request by jaebradley**

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -8,13 +8,10 @@ module.exports = {
         "devDependencies": [
           "config/*.js",
           "**/*.test.jsx",
-          "**/*.test.js",
+          "**/*.test.js"
```
Comment (thallada) [inline .eslintrc]: Pretty sure this could just be one line: `**/*.test.jsx?`

File: .travis.yml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 language: node_js
 node_js:
-- 8.9.4
+- 8.9.3
```
Comment (thallada) [inline .travis.yml]: Why the downgrade?

File: .eslintrc (hunk 1 above)
Comment (jaebradley) [reply .eslintrc]: I tried that but it didn't like it - I'll try again

File: .travis.yml (hunk 2 above)
Comment (jaebradley) [reply .travis.yml]: As we discussed on HipChat, I think this should be the min version of `node` at `edX`.

**Pull request merged**